### PR TITLE
apicast-mapping-rules.t conversion to APIcast::Blackbox

### LINKED ISSUE #1365: apicast-mapping-rules.t

### PR DESCRIPTION
**Pull request by tkan145**
Fixes #1365

### CODE REVIEW

File: t/apicast-mapping-rules.t [hunk 1]
```diff
@@ -1,139 +1,130 @@
 use lib 't';
-use Test::APIcast 'no_plan';
+use Test::APIcast::Blackbox 'no_plan';
 
 run_tests();
 
 __DATA__
 
 === TEST 1: call to backend is cached
 First call is done synchronously and the second out of band.
```
Comment (eguzki) [inline t/apicast-mapping-rules.t:9]: This test comment seems to me wrong

File: t/apicast-mapping-rules.t [hunk 2]
```diff
@@ -1,139 +1,130 @@
...
   location /api-backend/ {
-     echo 'yay, api backend';
+    echo 'yay, api backend';
   }
 --- request
 GET /staging/video/encode?size=100&speed=3x&user_key=foo&speed=2x
 --- response_body
 yay, api backend
 --- error_code: 200
+--- no_error_log
```
Comment (eguzki) [inline t/apicast-mapping-rules.t:46]:
```
--- no_error_log
[error]
``

**Review by eguzki — CHANGES_REQUESTED** [review]
The PR looking good.

minor issues

File: t/apicast-mapping-rules.t (hunk 2 above)
Comment (tkan145) [reply t/apicast-mapping-rules.t:45]: Fixed

File: t/apicast-mapping-rules.t (hunk 1 above)
Comment (tkan145) [reply t/apicast-mapping-rules.t]: Fixed

**Review by eguzki — APPROVED** [review]

**Pull request merged**